runner.yml.dist

```diff
@@ -0,0 +1,61 @@
+drupal:
+  root: "build"
+  base_url: "http://web:8080/build"
+  site:
+    profile: "minimal"
+    name: "OpenEuropa"
+  database:
+    host: "mysql"
+    name: "rdf_skos"
+    user: "root"
+    password: ""
+  sparql:
+    host: "sparql"
+    port: "8890"
+  post_install:
+    # Prepare the instance.
+    - "./vendor/bin/drush en toolbar -y"
+    - "./vendor/bin/drush en config_devel -y"
+    - "./vendor/bin/drush en field_ui -y"
+    - "./vendor/bin/drush theme:enable bartik -y"
+    - "./vendor/bin/drush theme:enable seven -y"
+    - "./vendor/bin/drush config-set system.theme default bartik -y"
+    - "./vendor/bin/drush config-set system.theme admin seven -y"
+    - "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y"
+    # Enable the modules.
+    - "./vendor/bin/drush en rdf_skos -y"
+  settings:
+    settings:
+      file_scan_ignore_directories:
+        - "node_modules"
+        - "bower_components"
+        - "vendor"
+        - "${drupal.root}"
+    databases:
+      sparql_default:
+        default:
+          prefix: ""
+          host: ${drupal.sparql.host}
+          port: ${drupal.sparql.port}
+          namespace: 'Drupal\Driver\Database\sparql'
+          driver: 'sparql'
+
+commands:
+  drupal:site-setup:
+    - { task: "symlink", from: "../../..", to: "${drupal.root}/modules/custom/rdf_skos" }
+    - { task: "run", command: "drupal:drush-setup" }
+    - { task: "run", command: "drupal:settings-setup" }
+    - { task: "run", command: "setup:phpunit" }
+    - { task: "run", command: "setup:behat" }
+  setup:phpunit:
+    - { task: "process", source: "phpunit.xml.dist", destination: "phpunit.xml" }
+  setup:behat:
+    - { task: "process", source: "behat.yml.dist", destination: "behat.yml" }
+  # Use the following commands only if you can run Task Runner commands on your host machine.
+  sparql:purge:
+    - "docker-compose exec sparql ./vendor/bin/robo purge"
+  sparql:import:
```

Comment:
`sparql:purge` and `sparql:import` are not very useful and it's not very coherent with the dev environment that we have. We should not use docker-compose commands.

Reply:
What is useless is to create two custom commands in the task-runner that we can use only if you run them outside the dockers, I am not talking concerning the commands it-self.